projects/baby_driver/scripts/repl_setup.py

```diff
@@ -0,0 +1,29 @@
+"""
+This module is run just before the Python REPL opens when you run `make babydriver`.
+It imports everything that's visible to the REPL by default and sets up the default CAN channel.
+"""
+
+# pylint: disable=unused-import
+from can_util import get_bus, send_message, next_message
+from message_defs import BABYDRIVER_DEVICE_ID, BABYDRIVER_CAN_MESSAGE_ID, BabydriverMessageId
```

Comment:
shouldn’t these be abstracted into the driver APIs? when would you need to use them directly? not super sure but haven’t thought about it much